workflow/scripts/pass_time_cylc.py

```diff
@@ -43,20 +44,16 @@ def __init___(self, args):
 def get_passtimes(
     startdate, enddate, csvoutpath, centroid_x, centroid_y, SPACEUSER, SPACEPSWD
 ):
-    centroidx = centroid_x
-    centroidy = centroid_y
-    configUsr = SPACEUSER
-    configPwd = SPACEPSWD
+    enddate = os.environ.get("enddate")
+    startdate = os.environ.get("startdate")
+    centroidx = os.environ.get("centroid_x")
+    centroidy = os.environ.get("centroid_y")
+    configUsr = os.environ.get("SPACEUSER")
+    configPwd = os.environ.get("SPACEPSWD")
+    csvoutpath = "/tmp"
```

Comment:
I see these args being passed to the function. Why are they being redefined here? If you prefer reading the env variables, then the function args are not needed. I think it's better to have the args, however.

Reply:
I was initially having trouble getting the args to pass in. But I have it worked out now. When running Docker directly (local), the credentials are passed as env vars, but for the script they need to be passed the same as other args with a $ in front, but also declared with --env for docker.